app/component/TicketInformation.js

```diff
@@ -81,7 +83,11 @@ export default function TicketInformation(
               <a href={config.ticketLink}>
                 <div className="ticket-identifier">
                   {config.useTicketIcons
-                    ? renderZoneTicket(fare.ticketName, alternativeFares)
+                    ? renderZoneTicket(
+                        // fare.product.id.split(':')[1],
```

Comment:
what is the purpose of this comment?

Reply:
This is again probably the result of forgetting to clean up after doing some experimentation during development. Removed these comments as unnecessary.